domains/arkaive19.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "arkaive19",
+        "email": "arkaive19@gmail.com"
+    },
+    "record": {
```

```suggestion
    "records": {
```
invalid JSON content please read more -> https://docs.is-a.dev/domain-structure/